featuretools_tsfresh_primitives/primitives/fft_aggregated.py

```diff
@@ -30,3 +35,115 @@ def function(x):
             return list(fft_aggregated(x, param=param))[0][1]
 
         return function
+
+
+class ShortTermFftAggregated(TransformPrimitive):
+    """Calculates the spectral centroid (mean), variance, skew, and kurtosis of
+    the absolute fourier transform spectrum of entries over a given window.
+
+    Description:
+        Given a list of numbers and a corresponding list of
+        datetimes, return the spectral centroid (mean), variance, skew, and kurtosis of
```

Comment:
Another use of `and` that might be better as `or`?

Reply:
Makes sense, I'll update this in the original primitive as well.

Reply:
Minor nitpick, but L46 still says `and` instead of `or`.

Reply:
my bad. resolved